SU-SWS/decanter
# D8CORE-1334: adjusting the padding to make the text appear centered

# READY FOR REVIEW

# Summary
- Adjusting the padding around the web login text in the button

# Needed By (Date)
- 2/21

# Urgency
- Low

# Steps to Test

1. Pull in the branch.
2. Have your testing site refer to this version of Decanter.
3. Build and clear caches.
4. Look at the bottom Web Login in button and notice it appears centered.

# Affected Projects or Products
- Effect v1.0.0 of the Core D8 project

# Associated Issues and/or People
- D8CORE-1334
- @kerri-augenstein 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
